app/controllers/posts_controller.rb

```diff
@@ -25,6 +26,7 @@ def create
   end
     
   def destroy
+     @post = :Post.find(params[:id])
```

Comment:
This is unnecessary. See [this](https://github.com/ks927/odin_facebook/pull/1/files#diff-ab4e9273f7df75aa5004a42f757d6f51R45) line of code.